packages/mediacenter/kodi/package.mk

```diff
@@ -345,4 +345,8 @@ post_install() {
   enable_service kodi-waitonnetwork.service
   enable_service kodi.service
   enable_service kodi-lirc-suspend.service
+
+  if [ $PROJECT = "S905" ]; then
+    enable_service kodi-aml-hdmimonitor.service
```

Comment:
You do not need this if you also move `kodi-aml-hdmimonitor.service` to `$PROJECT/filesystem/usr/lib/systemd/system`, all `.service`-files in such project/device path is enabled by the LE build system

Reply:
I didn't know this, thanks for the tip